output/cloud/output.go

```diff
@@ -145,13 +145,13 @@ func newOutput(params output.Params) (*Output, error) {
 // validateRequiredSystemTags checks if all required tags are present.
 func validateRequiredSystemTags(scriptTags *metrics.SystemTagSet) error {
 	missingRequiredTags := []string{}
-	requiredTags := metrics.TagName |
+	requiredTags := metrics.SystemTagSet(metrics.TagURL | metrics.TagName |
```

Comment:
hmm :thinking: I don't remember a reason, so this is most likely a mistake, will investigate and remove it.

Reply:
resolved by the [latest force-push](https://github.com/grafana/k6/compare/7c90538db8b799df11022500cb81d91c4e223dc1..6ba6ff20da1b1bd48f5f680b610795852cf5ce02), along with another issue I forgot to fix in the PR (since it was discussed outside of it :disappointed: )